mixer/adapter/signalfx/config/config.proto

```diff
@@ -75,4 +130,23 @@ message Params {
     // This will be rounded to the nearest second and rounded values less than
     // one second are not valid. Defaults to 10 seconds if not specified.
     google.protobuf.Duration datapoint_interval = 4 [(gogoproto.nullable) = false, (gogoproto.stdduration) = true];
+
+    // Optional.  If set to true, metrics won't be sent (but trace spans will
+    // be unless otherwise disabled).
+    bool disable_metrics = 5;
```

Comment:
This is nit-picky, but in general it's better to avoid booleans that disable something when true. It's cleaner to have enable_metrics as a boolean and default it to true.